alkira/resource_alkira_service_pan.go

```diff
@@ -454,6 +454,7 @@ func generateServicePanRequest(d *schema.ResourceData, m interface{}) (*alkira.S
 
 	service := &alkira.ServicePan{
 		BillingTagIds:               convertTypeListToIntList(d.Get("billing_tag_ids").([]interface{})),
+		Bundle:                      d.Get("bundle").(string),
```

Comment:
Thanks! This is probably missing from the READ function as well? Could we fix that.